pkg/api/dashboard/dashboard.go

```diff
@@ -109,10 +109,12 @@ func getRows(db *sql.DB, table string, metricName string, selector ResourceSelec
 	orderBy := []string{"name", "time"}
 	if metricName == "cpu" {
 		query = "select sum(cpu), name, uid, time from %s "
-	} else {
-		//default to metricName == "memory/usage"
-		// metricName = "memory"
+	} else if metricName == "memory" {
 		query = "select sum(memory), name, uid, time from %s "
+	} else if metricName == "ephemeral_storage" {
```

Comment:
It seems to me that the metricName would be more appropriate - "storage"

Reply:
That is a different metric

Reply:
You are right, in my comment relied just on the title of the column

Reply:
as far as I know, `Storage` is _only_ used for persistent volumes - not for nodes and pods